src/cow-react/modules/advancedOrders/hooks/useParseNumberOfParts.ts

```diff
@@ -1,9 +1,6 @@
 import { useCallback } from 'react'
 import { useSetAtom } from 'jotai'
-import {
-  updateAdvancedOrdersSettingsAtom,
-  NumberOfPartsError,
-} from '@cow/modules/advancedOrders/state/advancedOrdersSettingsAtom'
+import { updateAdvancedOrdersSettingsAtom } from '@cow/modules/advancedOrders/state/advancedOrdersSettingsAtom'
 import { MIN_PARTS_NUMBER, MAX_PARTS_NUMBER } from '@src/constants'
 
 export function useParseNumberOfParts() {
```

Comment:
I think we should we must continue to follow the pattern like in `updateLimitOrdersAtom`.
And I guess we won't need this hook if we move parsing logic to `TradeNumberInput`, see: https://github.com/cowprotocol/cowswap/pull/2479/files#r1198591948

Reply:
Similar to my comment above https://github.com/cowprotocol/cowswap/pull/2479#discussion_r1198749702